boards/nxp/imx8mp_evk/imx8mp_evk_mimx8ml8_a53.yaml

```diff
@@ -13,10 +13,11 @@ toolchain:
   - cross-compile
 ram: 1024
 supported:
-  - uart
-  - net
+  - can
```

Comment:
Both `flexcan1` and `flexcan2` are disabled by default, so listing `can` as supported is incorrect.

Could one of the CAN controllers + transceiver be enabled by default? If yes, please remove the added test configuration in `tests/drivers/can/api`.

Reply:
I would like to remove can from supported list but not enable it by default, thanks.

Reply:
I think we need to enable device per requirement, for example, we only just want to run a hello world, why we need to enable flexcan in hello world application? considering the image size and running performance, the application is as good as smaller. I agree with you that it is easy for customer to run any usecase if we enable all the device by default,  so as you said if this is normally done in Zephyr, I would like to align with the other platforms.

Reply:
Updated, please review again, thanks @henrikbrixandersen

Reply:
Brix is the Maintainer of CAN bus.
Nothing can related will go in without his approval.